test/conformance/testing/include/uur/environment.h

```diff
@@ -17,7 +17,7 @@ struct PlatformEnvironment : ::testing::Environment {
 
     struct PlatformOptions {
         std::string platform_name;
-        unsigned long platforms_count = 0;
+        unsigned long platforms_count = 1;
```

Comment:
can we make use of the CMake option/variable in all tests, so the change in here is not required...?

Reply:
I have changed it.